internal/versions/versions.go

```diff
@@ -109,6 +109,12 @@ func (h *handler) V2ListSupportedOpenshiftVersions(ctx context.Context, params o
 				arch = common.DefaultCPUArchitecture
 			}
 
+			// In order to mark a specific version and architecture as supported we do not
+			// only need to have an available release image, but we need RHCOS image as well.
+			if _, err := h.GetOsImage(key, arch); err != nil {
+				continue
```

Comment:
Probably should log the err here as a warning. It would help users identify the issue in case of a missing OS image.

Reply:
Done. As debug because we definitely don't want to see those in SaaS. For on-prem users we anyway troubleshoot with debug